pages/index.js

```diff
@@ -50,62 +56,159 @@ export const getServerSideProps = async () => {
   };
 };
 
-function HomePage({
+const getStateData = async (state) => {
+  const shootingsByCounty = await universalAPIFetch(
+    `count/shootings/statecounty/${state}`
+  );
+  const shootingsOverTime = await universalAPIFetch(
+    `count/shootings/overtime/${state}`
+  );
+  const last20Items = await universalAPIFetch(`shootings/last20/${state}`);
+  const topPoliceDepartments = await universalAPIFetch(
+    `count/shootings/pd/${state}`
+  );
+
+  return {
+    shootingsByCounty,
+    shootingsOverTime,
+    topPoliceDepartments,
+    last20Items,
+  };
+};
+
+export const getServerSideProps = async () => {
+  const nationalData = await getNationalData();
+  return nationalData;
+};
+
+function Dashboard({
   shootingsByState,
-  shootingsPerCapita,
+  shootingsByGeo,
   shootingsOverTime,
   topPoliceDepartments,
   last20Items,
+  selectedState,
+  setSelectedState,
+  setIsPerCapita,
 }) {
-  const [isPerCapita, setIsPerCapita] = useState(false);
-
   return (
     <Container>
       <Head />
+      <Row>
+        <Col sm="auto">
+          <h2>Police Killings in 2020</h2>
+        </Col>
+        <Col sm="auto" style={filterColumnStyle}>
+          {selectedState ? (
+            <h4>
```

Comment:
I like the choice of a badge element here to show what state is selected! That said, Is this `h4` element just being used for the text treatment? Heading elements indicate certain things to assistive tech that should be kept in mind:

- Skipping heading levels (this would go from `h2` to `h4` can make navigation confusing for some assistive tech which lets a user jump around the page by heading
- This would include the badge content in a breakdown of the content, which could also be confusing since it's a control and not an area of the page.

I'd recommend removing the `h4` for now. I'm working on some style changes in another branch to fold in Material UI instead of Bootstrap, so maybe the style of that element will mesh better with what you had in mind.

Reply:
Yea, this was definitely done for expediency to get the text styling. I did not know that skipping heading levels could be problematic, thanks for sharing!